deploy/examples/object.yaml

```diff
@@ -70,6 +70,14 @@ spec:
               # Use <topologyKey: failure-domain.beta.kubernetes.io/zone> in k8s cluster if your cluster is v1.16 or lower
               # Use <topologyKey: topology.kubernetes.io/zone>  in k8s cluster is v1.17 or upper
               topologyKey: kubernetes.io/hostname
+    logging:
+      opsLogEnabled: false
+      # resources:
+      #   limits:
+      #     memory: "60Mi"
+      #   requests:
+      #     cpu: "100m"
+      #     memory: "60Mi"
```

Comment:
I want to talk about this API some more.

`logging` suggests to me that this is the place for all logging-related configs. That isn't a poor choice, but it is important for the context to come after.

`opsLogEnabled` being a separate config implies that we expect other types of logging settings in the future beyond ops logging.

`requests` being directly under `logging` implies that all logging settings are affected by `requests`, but what happens if we add more log settings that don't affect the ops log sidecar?

I don't think this API structure does the best job of leaving room for the ways these configs might be expanded in the future.

I know you have said that we recommend `rgw_ops_log_rados = false`, but I can foresee users wanting a control for this. Or what about `rgw_log_nonexistent_bucket`, `rgw_log_object_name`, `rgw_log_object_name_utc`? If users want to control all of those, then the config has to look like this 

```
logging:
   opsLogEnabled: true
   opsLogRados: true
   opsLogObjectName: true
   opsLogObjectNameUtc: true
   opsLogNonexistentBucket: true
   resources:
     # blah
```

This is a situation we want to avoid, having a flat hierarchy of configs that all have the same prefix. Instead, we want to have a config section `opsLog` suggest what all child configs underneath imply.

Taking all this in whole, with the resources included, I think we would be better served by changing `logging` to `opsLogSidecar`. `enabled` can be omitted entirely, and then `resources` clearly goes with the sidecar. If we want to add the other configs in the future, they can be like `logRados`, `logObjectName`, `logObjectNameUtc`, `logNonexistentBucket` without doing complex or non-ideal API changes.